kolibri/plugins/learn/assets/src/vue/side-nav/nav-item.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+
+  <div class='link-wrapper'>
+    <a class='link' v-link="vlink" @click='closeSearch' :class="{active: active}">
```

Comment:
Ah, I see the [comment below](https://github.com/learningequality/kolibri/pull/319/commits/a4aa4b927317b5e9fd6f66ce90582b4f759796fe#diff-9a18a9cca259b56041fa61a01a4fa2bfR34). Perhaps `@click` could be bound to a more generic "top-level-nav-item-clicked" type event, and the apps can listen for that as desired.